esrally/track/track.py

```diff
@@ -631,7 +632,7 @@ def from_hyphenated_string(cls, v):
             return OperationType.IndexStats
         elif v == "node-stats":
             return OperationType.NodeStats
-        elif v == "search":
+        elif v in ["search", "scroll-search", "paginated-search"]:
```

Comment:
I think this is leaking an implementation detail (one runner class for several operation types) into other parts of the code. How about we instead register the `Query` runner for all three operation types? See:

https://github.com/elastic/rally/blob/35b1fbab6cf35d9e41b6b3db50af1e3a4ce0e424/esrally/driver/runner.py#L45

Reply:
looks like i got halfway to doing that then forgot it was happening haha